mono/metadata/icall.c

```diff
@@ -9111,7 +9119,7 @@ mono_register_jit_icall_full (gconstpointer func, const char *name, MonoMethodSi
 
 	info = g_new0 (MonoJitICallInfo, 1);
 	
-	info->name = name;
+	info->name = g_strdup(name);
```

Comment:
Some codepaths to mono_register_jit_icall_full frees the string elsewhere and some do not. The duplicated string is freed in mono_icall_cleanup.